server/consumer.go

```diff
@@ -295,7 +295,7 @@ const (
 	JsDeleteWaitTimeDefault = 5 * time.Second
 	// JsFlowControlMaxPending specifies default pending bytes during flow control that can be
 	// outstanding.
-	JsFlowControlMaxPending = 16 * 1024 * 1024
+	JsFlowControlMaxPending = 32 * 1024 * 1024
```

Comment:
* Before was:
1MB and maxpb = limit / 16
* 19 days ago:
changed to 32MB *AND* changed maxpb = limit / 4
* 4 days ago:
changed to 16MB *AND* changed maxpb = limit / 16
* today:
changed back to 32MB but maxpb not changed... hard to follow..

Reply:
Nope that is ok and will leave as is. Thanks.

Reply:
Sorry, so are you saying that there is no need to change maxpb even if you update JSFlowConstrolMaxPending? If so, let me know and I will approve the PR.

Reply:
Correct, I want it to start at /16 or 2mb